### PR TITLE
[master] Add API to get total coin supply as uint long

### PR DESCRIPTION
**Pull request by chetan-zilliqa**
## Description

Please refer to the ticket details for a conversation regarding the requirement.
https://zilliqa-jira.atlassian.net/browse/ZIL-5121

`curl --request POST \
  --url http://chetan-master-api-2-l2api.dev.z7a.xyz/ \
  --header 'Content-Type: application/json' \
  --data '{
    "id": "1",
    "jsonrpc": "2.0",
    "method": "GetTotalCoinSupply",
    "params": [""]
}'`

Response:
`{
	"id": "1",
	"jsonrpc": "2.0",
	"result": "12600152995.523188406040"
}` 

`curl --request POST \
  --url http://chetan-master-api-2-l2api.dev.z7a.xyz/ \
  --header 'Content-Type: application/json' \
  --data '{
    "id": "1",
    "jsonrpc": "2.0",
    "method": "GetTotalCoinSupplyAsInt",
    "params": [""]
}'`

Response:
`{
	"id": "1",
	"jsonrpc": "2.0",
	"result": 12600152995
}`
## Backward Compatibility

- [ ] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [x] small-scale cloud test

### CODE REVIEW

File: src/libServer/LookupServer.cpp [hunk 1]
```diff
@@ -1393,18 +1396,24 @@ string LookupServer::GetTotalCoinSupply() {
...
   streamObj << std::fixed;
   streamObj << std::setprecision(12);
-  streamObj << ans;
-
+  streamObj << CalculateTotalSupply();
   return streamObj.str();
 }
 
+unsigned long LookupServer::GetTotalCoinSupplyAsInt() {
+  return CalculateTotalSupply().convert_to<unsigned long>();
```
Comment (JamesHinshelwood) [inline src/libServer/LookupServer.cpp:1414]: Will this truncate? Is that what we want?

**Review by JamesHinshelwood — APPROVED** [review]
LGTM

File: src/libServer/LookupServer.cpp (hunk 1 above)
Comment (chetan-zilliqa) [reply src/libServer/LookupServer.cpp]: Thank you. It was truncating before. Added rounding logic.

Comment (chetan-zilliqa) [reply src/libServer/LookupServer.cpp]: We want to round to the nearest integer.

**Review by JamesHinshelwood — APPROVED** [review]

**Pull request merged**